pkg/env/env.go

```diff
@@ -124,7 +124,7 @@ func IsCI() bool {
 // Otherwise it returns nil.
 func ValidateVars(vars ...string) error {
 	for _, v := range vars {
-		if _, found := os.LookupEnv(v); !found {
+		if v, found := os.LookupEnv(v); !found || v == "" {
```

Comment:
reuse the variable "v" here does not sound very good, sometimes it is dangerous and hard to debug, it have no problem in the case tho. it does not help for a good coding practice, how about change to another one?

Reply:
oh, it does have problem here, in case the env variable is set to "", the error handling cannot print out the actual variable name, instead it only prints out an "" as v has been re-assigned.